### ISSUE TITLE
Crosshair only showing vertical line, not horizontal.

### ISSUE THREAD

**Issue by L0ll094** [AbelHeinsbroek/chartjs-plugin-crosshair#73]
Hi! Running angular 10.0.8 and using your plugin just like the example you provided. For some reason, I cannot see the horizontal cross hair. Is there a setting for this that I have just missed?

**Comment by laurensV** [comment]
would also like this feature

**Comment by JG-QuarkNet** [comment]
As would I

**Comment by nkorent** [comment]
same. is this supported even?

**Comment by firebird631** [comment]
Yes it would be nice to have this. With a special case in sync because if the y scale are different you should not draw the horizontal line in others but only for the hovered chart.
